ts/src/pro/coinex.ts

```diff
@@ -711,23 +736,46 @@ export default class coinex extends coinexRest {
     }
 
     async watchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        /**
+         * @method
+         * @name coinex#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.coinex.com/api/v2/spot/order/ws/user-order
+         * @see https://docs.coinex.com/api/v2/futures/order/ws/user-order
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int} [since] the earliest time in ms to fetch orders for
+         * @param {int} [limit] the maximum number of order structures to retrieve
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {bool} [params.stop] if the orders to watch are stop orders or not
+         * @returns {object[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
         await this.loadMarkets ();
+        const stop = this.safeBool (params, 'stop');
+        params = this.omit (params, 'stop');
         await this.authenticate (params);
         let messageHash = 'orders';
```

Comment:
right now the messagehash does not contain the type when the `symbol` is not provided, so users can't subscribe to swap+spot orders simultaneously